MCprep_addon/spawner/mcmodel.py

```diff
@@ -461,15 +476,35 @@ def update_model_list(context: Context):
 
 		# Filter out models that can't spawn. Typically those that reference
 		# #fire or the likes in the file.
-		if "template" in name:
+		# These blocks just don't make sense to put in the for "unspawnable_for_now"
+		# Template base of that block for example candle, cake with candles
+		# Orient blocks base, cube same as orientable (no texture)
+		# Light blocks are just special no geometry block with 15 states of light levels
+		# Shulkers, Hanging Signs, Signs are entities, put it here for now since they have a lot of variants
+		# pitcher_crop_top_stage_1 is a top part of a double plant. I have no idea why it have no geometry.
```

Comment:
Formatting comment: I'd add a `-` in front of each item to make a bulleted list, just so it's easier to read